PolyEngine/Standalone/Windows/Src/Main.cpp

```diff
@@ -126,6 +128,7 @@ int WINAPI WinMain(HINSTANCE hInstance,
 				quitRequested = true;
 		}
 
+		quitRequested |= Engine.ShouldQuit();
```

Comment:
`|=` performs bitwise or and assign operation which is invalid in this context. Please replace it with `quitRequested = quitRequested || Engine.ShouldQuit();`. Also let's move this implementation after call to `Update`.

Reply:
One wonders why there is no corresponding `||=` operator, eh? 😉